front-end/src/pages/HomePage.tsx

```diff
@@ -1,17 +1,23 @@
 import axios from 'axios';
 import React, { useEffect, useState } from 'react';
+import ReactDOM from "react-dom"
 import { RouteComponentProps } from 'react-router-dom';
 import { Row, Col, Form, Input, Button } from 'reactstrap';
 import CreateEditPostModal from '../components/CreateEditPostModal';
 import PostListItem from '../components/PostListItem';
 import { Post } from '../types/Post';
-
+import AuthorList from "../components/AuthorList"
+interface userProps {
+  url: string;
+  displayName: string;
+  id: string;
+  github: string;
+}
```

Comment:
Please use the `Author` type (`import Author from "../components/Author"`)

You can see how to use it inside `AuthorPage.tsx`